homeassistant/generated/integrations.json

```diff
@@ -3585,6 +3585,13 @@
       "config_flow": true,
       "iot_class": "local_push"
     },
+    "mcp_server": {
+      "name": "Model Context Protocol Server",
+      "integration_type": "hub",
```

Comment:
I don't think it's a hub. We don't integrate any devices.

Reply:
Changed to service. hub was default since it was unset, whoops.